.travis.yml

```diff
@@ -1,23 +1,19 @@
-script: rake ci
+script: bundle exec rake
```

Comment:
The default has _more_ options. All of them good for us:

```shell
bundle install --jobs=3 --retry=3
```

https://docs.travis-ci.com/user/languages/ruby/#Travis-CI-uses-Bundler

I'll make this change!